CMSgov/mint-app
# Easi 3524/cm ffs counterpart role

# EASI-3524
## Changes and Description

- BE
   -  Migration to add the new team role
   - Update Schema
   - Update GO model
- Update Read View
    - FilteredViewGroupings takes a heading prompt
    - Update dfsdm filter view to show users in this role
    - add empty state for when there is now users assigned to the role. 



## How to test this change
1. Bring up the app.
2. Make a model plan
3. Add a collaborator with the new CM_FFS_Counterpart role
4. Verify that the role is persisting and is visible correctly in the read only view.


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
